js/modules/k6/http/request_test.go

```diff
@@ -129,9 +129,9 @@ func assertRequestMetricsEmittedSingle(t *testing.T, sampleContainer stats.Sampl
 	}
 }
 
-func newRuntime(
-	t testing.TB,
-) (*httpmultibin.HTTPMultiBin, *lib.State, chan stats.SampleContainer, *goja.Runtime, *context.Context) {
+func newRuntime(t testing.TB) (
+	*httpmultibin.HTTPMultiBin, *lib.State, chan stats.SampleContainer, *goja.Runtime, *ModuleInstance,
+) {
```

Comment:
Yeah, that would be much better, but I'd like to do it in another PR... This one is so short mostly because I was able to keep the same number of results to `newRuntime()` and most places just didn't use the context and had `_` in there :sweat_smile: